setup.py

```diff
@@ -1,33 +1,35 @@
-#!/usr/bin/env python
+from glob import glob
+from setuptools import setup
 
-from __future__ import print_function
+models = [c for c in glob('models/*pyrnn.gz')]
+scripts = [c for c in glob('ocropus-*') if '.' not in c and '~' not in c]
 
-import sys
-import glob
-import os.path
-from distutils.core import setup
-
-assert sys.version_info[0]==2 and sys.version_info[1]>=7,\
-    "you must install and use OCRopus with Python version 2.7 or later, but not Python 3.x"
-
-if not os.path.exists("models/en-default.pyrnn.gz"):
-    print()
-    print("You should download the default model 'en-default.pyrnn.gz'")
-    print("and put it into ./models.")
-    print()
-    print("Check https://github.com/tmbdev/ocropy for the location")
-    print("of model files.")
-    print()
+setup(
+    name='ocropy',
+    version='2.0.0a1',
```

Comment:
* As I said somewhere else before, I'm fine with `1.5` but since this might break code it would justify a major bump. It's just a number. We could also switch from semantic versioning to year-month versioning as many other projects have (e.g. `19.1`) but that would be pretty drastic.
* Dropping the `v` is the right thing to do. https://packaging.python.org/guides/distributing-packages-using-setuptools/#version